### DISCUSSION TITLE
Possibility for mutually recursive structures using this 'and' keyword within different files

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/fsharp#15483] [Ideas]
Today I am talking about, mutually recursive structures using this 'and' keyword is possible within the same module, but not if they are in different files + modules
A lot people create a file for a type so would be nice if we can automatically (like C#) or manually (via 'and) get to use the type we created in another file

**Comment by Happypig375** [comment]
Language suggestions go to https://github.com/fsharp/fslang-suggestions and not here. Closing as duplicate of https://github.com/fsharp/fslang-suggestions/discussions/1278
